bundles/core/org.openhab.core.library/META-INF/MANIFEST.MF

```diff
@@ -11,7 +11,8 @@ Bundle-License: http://www.eclipse.org/legal/epl-v10.html
 Bundle-SymbolicName: org.openhab.core.library
 Bundle-DocURL: http://www.openhab.org
 Originally-Created-By: Apache Maven Bundle Plugin
-Import-Package: org.openhab.core.events,
+Import-Package: org.apache.commons.lang;version="2.6.0",

```

Comment:
please remove the version unless you really would like to restrict your Bundle to use at least 2.6.0